python/lsst/ts/externalscripts/make_LOVE_stress_tests.py

```diff
@@ -0,0 +1,299 @@
+        """Authenticate on the LOVE-manager instance
+        to get an authorization token."""
+
+        url = f"http://{self.location}/manager/api/get-token/"
+        data = {
+            "username": self.username,
+            "password": self.password,
+        }
+        resp = requests.post(url, data=data)
+        try:
+            token = resp.json()["token"]
+        except Exception:
+            raise Exception("Authentication failed")
+        self.websocket_url = (
+            f"ws://{self.location}/manager/ws/subscription?token={token}"
+        )
+
+    async def handle_message_reception(self):
+        """Handles the reception of messages."""
+
+        if self.websocket:
+            async for message in self.websocket:
+                if message.type == aiohttp.WSMsgType.TEXT:
+                    msg = json.loads(message.data)
+                    if "category" not in msg or (
+                        "option" in msg and msg["option"] == "subscribe"
+                    ):
+                        continue
+                    self.received_messages = self.received_messages + 1
+                    tracing = msg["tracing"]
+                    tracing["client_rcv"] = Time.now().tai.datetime.timestamp()
+                    self.msg_traces.append(tracing)
+
+    async def subscribe_to(self, csc, salindex, stream, topic_type):
+        """Subscribes to the specified stream"""
+
+        subscribe_msg = {
+            "option": "subscribe",
+            "category": topic_type,
+            "csc": csc,
+            "salindex": salindex,
+            "stream": stream,
+        }
+        await self.websocket.send_str(json.dumps(subscribe_msg))
+
+    async def start_ws_client(self):
+        """Start client websocket connection"""
+
+        async with aiohttp.ClientSession() as session:
+            self.websocket = await session.ws_connect(self.websocket_url)
+            for csc in self.event_streams:
+                for stream in self.event_streams[csc]:
+                    await self.subscribe_to(
+                        csc.split(":")[0], csc.split(":")[1], stream, "event"
+                    )
+            for csc in self.telemetry_streams:
+                for stream in self.telemetry_streams[csc]:
+                    await self.subscribe_to(
+                        csc.split(":")[0], csc.split(":")[1], stream, "telemetry"
+                    )
+            await self.handle_message_reception()
+
+
+class StressLOVE(salobj.BaseScript):
+    """Run a stress test for one or more CSCs.
+
+    Notes
+    -----
+    **Details**
+
+    * Run a LOVE stress test by generating several client connections
+    that will listen to every event and telemetry of the specified CSCs
+    """
+
+    def __init__(self, index):
+        super().__init__(index=index, descr="Run a stress test for one or more CSCs")
+
+        # number of clients to simulate on the stress test
+        self.number_of_clients = 10
```

Comment:
I suggest making this a configuration parameter for the script. It can have a default value, if you like.

Also please consider adding a parameter to control the data rate (e.g. event_interval or something similar).